fe1-web/src/features/evoting/network/ElectionHandler.ts

```diff
@@ -70,7 +70,8 @@ export const handleElectionOpenMessage =
     console.log('Handling Election open message');
     if (
       msg.messageData.object !== ObjectType.ELECTION ||
-      msg.messageData.action !== ActionType.OPEN
+      msg.messageData.action !== ActionType.OPEN ||
+      !msg.laoId
     ) {
       console.warn('handleElectionOpenMessage was called to process an unsupported message', msg);
```

Comment:
I'm not sure this error message is very clear in the case where `msg.laoId` is `undefined`. Maybe another if printing a different error message could be nice, but I'm not sure if a message can be sent without `laoId`.
The same applies for all error in this file.